### DISCUSSION TITLE
Github asking school issued email, but my college doesn't provide any academic email to students.

### DISCUSSION THREAD

**Discussion by KaushikKundu** [community/community#75548] [GitHub Education]
### Hi everyone,

I'm a college student from India studying computer science engg. I tried to apply for the student developer pack but it's asking for my academic email address provided by the college. Unfortunately, my college doesn't provide email ID to students, it's only for teachers. I've my college-verified id card and other documents to proof, but not verified academic email.
Without academic email, I can't even proceed and show other documents. Please let me know how to solve this. 
<img width="516" alt="image" src="https://github.com/community/community/assets/105000989/e8cade30-ee18-4f91-a03f-755c5a1329fc">

**Comment by AidenJamesYt** [comment]
for the school issued email, you **need** to have a email from your school ending with the handle @gni.edu.in (ex: yourname@gni.edu.in)

you need to have an academic email either way. try to see if your school can grant you one

**Reply by KaushikKundu** [reply]
But college doesn't provide email to students. What should I do ?

**Reply by AidenJamesYt** [reply]
Okay, so, to apply for the GitHub Student Developer Pack without a verified academic email address, you can follow these steps:

Create a GitHub account: If you don't already have a GitHub account, you'll need to create one. You can sign up for a free account using your personal email address.

Go to the GitHub Student Developer Pack page: Once you have a GitHub account, you can visit the GitHub Student Developer Pack page.

Click on the "Get your Pack" button: This will start the verification process.

Select "Manual verification" option: Since you don't have a verified academic email address, you'll need to select the "Manual verification" option.

Fill out the form: Provide your name, school name, and a brief description of how you plan to use the GitHub Student Developer Pack.

Upload your documentation: Upload a clear and legible copy of your college-verified ID card or other documents that prove your current enrollment.

Submit your application: Once you've uploaded your documentation, click on the "Submit" button to submit your application.

GitHub will review your application and respond within 2-3 business days. If your application is approved, you will receive an email with instructions on how to activate your GitHub Student Developer Pack.

**Reply by Codes-World** [reply]
Where is The "Get Your Pack" Button??
Please tell me if i go to GitHub Student Developer Pack page. I was only see the "Yes, I'm a Student" button only.

**Reply by KaushikKundu** [reply]
Click on the button. You should see sign up for student developer pack button.

**Reply by Codes-World** [reply]
Where is the Button?? Please let me know..

**Reply by Codes-World** [reply]
Yes, I see sign up for student developer pack button..

**Reply by Codes-World** [reply]
But After that go to the application page where we need to school email id and i don't have any school email id.

**Reply by Codes-World** [reply]
![Uploading Capture.PNG…]()

**Reply by KaushikKundu** [reply]
That's the problem I had. Sadly I don't have any ans to that. I'm stuck too.

**Reply by Codes-World** [reply]
Bro, I got the GitHub Student Developer Pack...

**Reply by YousePhi** [reply]
how ? I have the same problem

**Reply by KaushikKundu** [reply]
Can you explain the procedure?
